src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final int DIVIDER = 2;
+    private double height;
+    private double firstSide;
+    private double secondSide;
+
+    public IsoscelesTrapezoid(String color, double height, double firstSide, double secondSide) {
+        super(color);
+        this.height = height;
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstSide + secondSide) * height / DIVIDER;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Shape:\t" + FigureName.ISOSCELES_TRAPEZOID
+                + ",\tColor:\t" + this.getColor()
+                + ",\tArea:\t" + this.getArea()
+                + ",\tFirst Side:\t" + this.getFirstSide()
+                + ",\tSecond Side:\t" + this.getSecondSide()
+                + ",\tHeight:\t" + this.getHeight());
```

Comment:
Use name() for getting String representation of enum constants. Replace `FigureName.ISOSCELES_TRAPEZOID` with `FigureName.ISOSCELES_TRAPEZOID.name()` for consistency and to prevent issues if `toString()` is overridden in the enum.